src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="product" data-qa="card">
+      <img src="images/imac.jpeg"
+          alt="Imac"
+          class="product__image"
+        >
```

Comment:
if you are placing attributes on separate lines, it would be better to do this for all attributes. It is more clear for readability
```suggestion
      <img
        src="images/imac.jpeg"
        alt="Imac"
        class="product__image"
      >
```